client.py

```diff
@@ -1,252 +1,144 @@
+"""
+ Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ 
+ Licensed under the Apache License, Version 2.0 (the "License")
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ """
+
+from itertools import count
 import logging
-import random
+import os
+from unicodedata import numeric
 
 import hazelcast
-from hazelcast.core import HazelcastJsonValue
 from hazelcast.discovery import HazelcastCloudDiscovery
+from hazelcast.serialization.api import (
+    CompactSerializer,
+    CompactWriter,
+    CompactReader,
+)
 
 """
-This is boilerplate application that configures client to connect Hazelcast Cloud cluster.
+A sample application that configures a client to connect to a Hazelcast Viridian cluster,
+and to then insert and fetch data with SQL, thus testing that the connection to
+the Hazelcast Viridian cluster is successful.
 
-See: https://docs.hazelcast.com/cloud/python-client
+See: https://docs.hazelcast.com/cloud/get-started
 """
 
+class City:
+    def __init__(self, country:str, city:str, population:int) -> None:
+        self.country = country
+        self.city = city
+        self.population = population
 
-def city(country: str, name: str, population: int) -> HazelcastJsonValue:
-    return HazelcastJsonValue({
-        "country": country,
-        "city": name,
-        "population": population,
-    })
 
+class CitySeriazlizer(CompactSerializer[City]):
+    def read(self, reader:CompactReader):
```

Comment:
missing return values on various methods, can you add those? (You can install https://pypi.org/project/mypy/ and run it on strict mode)

Reply:
Sure, I have sent the following PR which includes some styling fixes https://github.com/emreyigit/hazelcast-cloud-python-sample-client/pull/1

Reply:
Thanks, merged.